jenkinsapi/build.py

```diff
@@ -531,3 +532,16 @@ def get_env_vars(self):
                           'is installed.')
             raise ex
         return data['envMap']
+
+    def toggle_keep_build_forever(self):
+        """
+        Toggles the keep this build forever flag for the specified build.
+        """
+        requester = self.job.jenkins.requester
+        if not isinstance(requester, CrumbRequester):
```

Comment:
Please leave `requester` as is - not all people use it and if you set it here, this will break on Jenkins where CSRF protection is switched off.